soc/renesas/smartbond/da1469x/power.c

```diff
@@ -49,6 +50,12 @@ static int renesas_da1469x_pm_init(void)
 
 	da1469x_sleep_config(&sleep_cfg);
 
+	/*
+	 * Give PD_SYS control back to PDC.
+	 * Control was acquired during PRE_KERNEL_1.
+	 */
+	da1469x_pd_release_nowait(MCU_PD_DOMAIN_SYS);
```

Comment:
Based on the `da1469x_sleep` HAL API, releasing PD_SYS here should not be correct, as PD_SYS will eventually be released begore calling `da1469x_enter_sleep` and be acquired again upon wakeup. In addition, `da1469x_sleep` will check if `da1469x_pd_release_nowait` returns a positive number which implies that the SoC should enter the WFI state and not the normal extended sleep state (a peripheral block powered by PD_SYS is actively used).

Reply:
Removed